src/tz.cpp

```diff
@@ -697,12 +699,14 @@ load_timezone_mappings_from_xml_file(const std::string& input_path)
         }
     }
 
-    is.close();
+    //is.close(); Our stream cannot be closed, because it is not a file stream. 
     return mappings;
 }
 
-#endif  // _WIN32
+#endif  // _WIN32 
+#endif //!USE_OS_TZDB
 
+#if !USE_OS_TZDB
 // Parsing helpers
```

Comment:
This code begins including Uncompiled TZDB parsing code, which conflicts with our codepath